pkg/scheduler/algorithm/priorities/selector_spreading.go

```diff
@@ -220,11 +204,13 @@ func (s *ServiceAntiAffinity) CalculateAntiAffinityPriorityMap(pod *v1.Pod, meta
 		firstServiceSelector = getFirstServiceSelector(pod, s.serviceLister)
 	}
 	//pods matched namespace,selector on current node
-	var selectors []labels.Selector
+	var selector labels.Selector
```

Comment:
space at the beginning of the comment above.

Reply:
Done